meta/ee-bindep.txt

```diff
@@ -11,4 +11,5 @@ openssl [platform:rpm]
 python3-cryptography [platform:dpkg]
 python3-cryptography [platform:rpm]
 python3-openssl [platform:dpkg]
-python3-pyOpenSSL [platform:rpm]
+python3-pyOpenSSL [platform:rpm !platform:rhel-9 !platform:centos-9]
+python3-pyOpenSSL [platform:rhel-9 platform:centos-9 epel]
```

Comment:
How about the following:
```suggestion
# On RHEL 9+ and CentOS Stream 9+, python3-pyOpenSSL is part of EPEL
python3-pyOpenSSL [platform:rpm !platform:rhel !platform:centos]
python3-pyOpenSSL [platform:rhel-6 platform:centos-6 platform:rhel-7 platform:centos-7 platform:rhel-8 platform:centos-8]
python3-pyOpenSSL [platform:rhel platform:centos !platform:rhel-6 !platform:centos-6 !platform:rhel-7 !platform:centos-7 !platform:rhel-8 !platform:centos-8 epel]
```
1. For non-RHEL/CentOS RPM-based systems (like OpenSuSE), `python3-pyOpenSSL` from regular packages will be used;
2. For RHEL/CentOS 6, 7, and 8, `python3-pyOpenSSL` from regular packages will be used;
3. For RHEL/CentOS other than 6, 7, and 8, `python3-pyOpenSSL` from `epel` will be used.